### ISSUE TITLE
Batch GMM does not update the mean and variance within each iteration

### ISSUE THREAD

**Issue by TechNIN9** [smiler96/GMM-KMeans-PyTorch#2]
Is it okay for the Batch GMM not to update the μ and variance within each iteration?I see that you comment the codes that updates  μ and var within each iteration
`# self.mu[k] = torch.sum(x * prob_k, dim=0) / self.alpha[k]`
`# self.var[k] = torch.mm(delta_t, delta) / self.alpha[k] + self.eps_mat`
Is there some problem with this usage?

**Comment by TechNIN9** [comment]
Sorry I didn't notice that the update process has already been in the fit function😂
